src/main/app/src/Components/Landscape/Search/Search.tsx

```diff
@@ -73,20 +73,22 @@ const Search: React.FC<PropsInterface> = ({locateFunction, setSidebarContent, ..
             .replace(/\*/g, '%2A')
       ).then((result) => {
         setResults(result);
-        const searchResult = results.map((value1) => (
-          <Item
+      });
+    },
+    []
+  );
+
+  useEffect(() => {
+    const searchResult = results.map((value1) => (
+        <Item
             small={true}
             key={value1.fullyQualifiedIdentifier}
             useItem={value1}
             locateItem={locateFunction}
-          />
-        ));
-        setSidebarContent(searchResult);
-        setHasChange(false);
-      });
-    },
-    [results, setSidebarContent, locateFunction]
-  );
+        />
+    ));
+    setSidebarContent(searchResult);
+  }, [results, setSidebarContent, locateFunction]);
```

Comment:
`setResults(result)` triggers a state change in the `Search` component and provides the new `results` to the component after a new render.

` const searchResult = results.map()` operates on the `results` available in the component. Since the new `results` are only available on the new render, we lagged behind one input character.

Now, located on an own `useEffect()`, the `searchResult` is passed on to the `setSidebarContent` function correctly, which triggers again a new render.

Reply:
excellent 👍